### ISSUE TITLE
Bug report: Application crashed with TraitError

### ISSUE THREAD

**Issue by mbercx** [aiidalab/aiidalab-qe#188]
## Automated report

_This issue was created with the app's automated bug reporting feature.
Attached to this issue is the full traceback as well as an environment
fingerprint that contains information about the operating system as well as all
installed libraries._

## Additional comments (optional):

Trying to select "Open Materials Database" from the OPTIMADE client results in this error.

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py in set(self, obj, value)
    578 
    579     def set(self, obj, value):
--> 580         new_value = self._validate(obj, value)
    581         try:
    582             old_value = obj._trait_values[self.name]

/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py in _validate(self, obj, value)
    612             value = self.validate(obj, value)
    613         if obj._cross_validation_lock is False:
--> 614             value = self._cross_validate(obj, value)
    615         return value
    616 

/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py in _cross_validate(self, obj, value)
    618         if self.name in obj._trait_validators:
    619             proposal = Bunch({'trait': self, 'value': value, 'owner': obj})
--> 620             value = obj._trait_validators[self.name](obj, proposal)
    621         elif hasattr(obj, '_%s_validate' % self.name):
    622             meth_name = '_%s_validate' % self.name

/opt/conda/lib/python3.7/site-packages/traitlets/traitlets.py in __call__(self, *args, **kwargs)
    975         """Pass `*args` and `**kwargs` to the handler's function if it exists."""
    976         if hasattr(self, 'func'):
--> 977             return self.func(*args, **kwargs)
    978         else:
    979             return self._init_call(*args, **kwargs)

/opt/conda/lib/python3.7/site-packages/ipywidgets/widgets/widget_selection.py in _validate_index(self, proposal)
    223             return proposal.value
    224         else:
--> 225             raise TraitError('Invalid selection: index out of bounds')
    226 
    227     @observe('index')

TraitError: Invalid selection: index out of bounds
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNpdWNtu4zgS_ZXA-zoyLNlx7LwNsk-L7tneTfcsFoPBgKZomzZFMiQVtzLo_fatmxRlkLzo8FasOnWq6D8XryZlG_zisf5pEZ0q
x5C6xeOfC5X02RajS5_M4vG3xXZzsGXx02LxO8wbyjn4P6ali_XyYbmHwXfkb_Wq-V-9W642y7r6duh96e-eP3-5-3ru734Jr3d1
fVdvHjfbx_X27tvXp7tm1dSLH7C10ld1MplMsLZVlQ5owKJebpf3cASDMZu-
DQCvAF5P8Euv4KDO5JhMzoEsu1_W47hTB4CaelnXH8DKOE97_QV-MdP01Ue8uoV01WdlPZn2YfRm25MpuTqozIavZTxU0V4VQNvl
boRS90JWrvlk115MN5Ate0FMd7Ca9nlgB_hsm3PpnDiFNoqxtSkTslluEMHwvZoqqnKm7RreLp106KIzha62XPHydAq-qfTxaPnC
67_C1cH61vpT5vFGxlO40ZmyOd0XbrLnkOTB6-pkvEmqhEQTa1pXSvqw0QGCrpVzYihBOg2x0HYCWK_SEJIPhabxNQ_OKI0XBJrx
tHA1Z7raZtng90t0sqJulms085BCcTaylx9o1RNsYp6CLym4cS6ZqnGghOAyndHQodqkYtlVK2THarlDlL0H32wasCO1ptA6djMi
2ZTKQ9SVs28mSQz4qLPprD-GD4Zpm3Sf2aQHcrJ2Vl8BgC-6IH1XElTOvRVwfj7kj_ZUHa0zki-raSxH6z2ZQdyngeBbRUbveRP8
riQtq7PyrQMeCB9lAuwfDhfA7uEyW8LC1Rrdl2KSTEVGUEzDKal4Hmas14N2oxEcxVZpW8yM4K059CfyTC0J3RoQBuHVvQS_Ncc-
m_Z758SFPBHEQCvxTSNT_35R_hQIgD-MV-sz69psVtu7m9WSQPCPcTaQkcYXSSzkGc40nbKueoWwtu9kp0vDXLh2sL5wHAkrZs2p
Qi48qlxUtByee5qB4XKBIo1uolnAzxkVdwz2LNDoO0bIva_2jbE9IcZ4R1Ss5UTbesX3RONtF0OCnDhUnSkKLsAEqJm276MgqqFP
GtUZfN6QoZBIuEBET7JkxQNXk7xxJHhbioWdPAyEWI3TEEKd6It1eaYBOASmz-QLkPeDHqQivIOV-V6Mb00rN8UlF9Pa0T34_Q_r
L6rhu3OALoRUxXZmdvglHODKshEBOXgKo6RtI1gyR14m3xkTWc2U69LDPUyqIOWAC-QN2LOZj3CJ24jmCzyrJVxk6Iirvb1r10aA
HNyrZBpzxal0haQFuUmjnqEpyKhJCGq-w2d1DXLNLX2maxtuXog3Qn18VsexZqCRnSqorOwjrLKrD2BlPeiEHERu7mwuvSA7srwD
Qg-jikJ0HghLpoLcTyPTd6PtXew45cZy0-UTypL4ho7oXbGt1UX4icv8AQQKnMMY09AfsMtRRaQDl3qTS0UVywYpowwXrPTf6eLc
afiTe7XmRrfmY6EkmQMoHoV2Q5z0fSdqBVdDIDgzOX6DTg0RCKda8dCWSsiIvVNFXEMCxRrM4tuw1RHag1LUwRkJ14rA7iWSweM3
KHgWctwLEDSYA27OIqk8ManOXlkVOVuQQkFC1tB3OY-RWYmWxANrMPc00XyPRnPZE-SU-1kORyg8zmSohUZYjDZ9sc5RM7ETQf1i
fRl9w8vi2JvQJ9JsoHN5fnTiclJqQsIQpBrSZwqgbmfT53f3TomBg7FUeLWrLRLbhlycUZbmF8yDDvHUVNyRiLMwpLEMsA_IY55V
8DhoZRNzquHU-
TI82eO__0MuYprBpJBVmfWzgEzpC3nKO7XQ3UovuGcnDSesRVmkXA4EZoOaMnsoXbc09fPw_K9P0lquCflFPTkhO6_8ZzT--fkTL
ZU8jwPawaxbS3mPAyC5TD4Uxwy5A4qPYsU3fQbFydIC8Hak9lAyjPi1GanFI9n1J3scpJkYB6AMXPKc00MJF626KDwULClmwHpc-
EZXqccL__fnz5-EM2zMG7XfzbgtNPYju6k3Thr3wfJZ0jCT12SoJKpR-7Y7Al96UBFe3HDfMmIVNZKz3mwa6IJIGDsxHfV6v3uQs
2jbXnXGLQclXQ3mw_ojvtQsw5hizTRU4VA19nOwEBiMR2RtgeMVdM6Ju8WGW2XARbO4W8vGXOMoufslqvMzVNfma1L5PDMZWto-
To3JHju2ZQy5oHat6tXuw5Qq6450kh2Zz8ZhP3lW3YyI-Qze7XvbCpVoC4sdLhZYbpweiInZfufaxTme42kq2ls6ACj_s8OKPIVv
8wGtvk1Nx5rbifziKPPklUFWFpVQaEWsRzBBpakwYOv1njMe4WFUfmy9QJt7eFSPVY8g47G7HUjrOE1Ah6H1V_KgrRsBc5leb3y9
Av1N1XuLvW8rb0uEIT1UG8G9ujhJU5oeRspwIuG3lQLK30kOZYktL21HonQv30nZ4rj9uJd2ijDQOZNnkg7fEMM_qPvC93-WRw-N
vUHToTjR8bv3-HyCh3OxtMda-o4-HiH3KgnwqFF9Anoc1mw0K-OvCnJvmNWfmyr63IbTeFUM8E1D5zS9fDGsNwONgAtS8OQHAACN
B1_Ky3Ylb4vb2Rg3coKOoDYMXvS-
zdN7UvbloWiSDdh6SDHGF8T7cPaHyTkiGbQttOpltAcBeLlM7WMtD7ABuDd7br3ZGGkLFJgfP_4PPA77xw==</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

**Comment by rartino** [comment]
Can you just retry it? We had some downtime back in January/February, so I really think this was just a temporary outage.

**Comment by rartino** [comment]
@unkcpz Yes - sorry - we didn't use to return coordinates (just like cod...), but based on your report I actually decided to upgrade our OPTIMADE setup to do this. Nevertheless, you seem to be right that species is not there. I'll ping you here when I think our setup is ready.
